modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java

```diff
@@ -320,6 +319,8 @@ public void testUpdateDefaultObjectStateFlow() throws Exception {
 
 	@Test
 	public void testUpdateObjectStateTransitions() throws Exception {
+		_step4ListTypeEntry = _addListTypeEntry("step4");
+
```

Comment:
@murilostodolni can we make it local? As we are only using it in this test.

```
ListTypeEntry listTypeEntry = _addListTypeEntry("step4");
```

Reply:
Yes! Very well observed. Thanks